standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/datasource/HikariCPDataSourceProvider.java

```diff
@@ -52,10 +50,7 @@ public DataSource create(Configuration hdpConfig, int maxPoolSize) throws SQLExc
     String user = DataSourceProvider.getMetastoreJdbcUser(hdpConfig);
     String passwd = DataSourceProvider.getMetastoreJdbcPasswd(hdpConfig);
 
-    Properties properties = replacePrefix(
-        DataSourceProvider.getPrefixedProperties(hdpConfig, HIKARI));
-    long connectionTimeout = hdpConfig.getLong(CONNECTION_TIMEOUT_PROPERTY, 30000L);
-    long leakDetectionThreshold = hdpConfig.getLong(LEAK_DETECTION_THRESHOLD, 3600000L);
```

Comment:
As a new unit test explains, the default connection timeout is originally 30 seconds.
We also decided to use the default leak detection threshold. It doesn't change the behavior because the current value is invalid, and the detection is disabled.